docs/paying-for-lit/overview.md

```diff
@@ -0,0 +1,60 @@
+# Paying for the Usage of the Lit Network
+
+Like other decentralized networks, Lit has a certain amount of computation available for users that's metered to allow for a responsive network with nodes that are able to stay in-sync with one another.
+
+In order to use the paid pre-production ([Datil-test](../connecting-to-a-lit-network/testnets#datil-test)) and production ([Datil](../connecting-to-a-lit-network/mainnets#datil)) Lit networks, you must reserve capacity on the network. This is done using by minting Capacity Credits, and by making use of the Lit Relayer and/or the Payment Delegation Database.
+
+## Overview of Payment Methods
+
+### Capacity Credits
+
+Capacity Credits are NFT tokens that represent reserved computational capacity on the Lit network. When minting a credit, you specify and pay to reserve a specific amount of requests per second over a period of time. You'll then use these credits when making requests to the Lit network to perform actions such as decryption, executing Lit Actions, and using PKPs.
+
+For a deep dive into Capacity Credits, including minting and usage details, checkout the in-depth [documentation](./capacity-credit-intro.md).
+
+### Lit Relayer
+
+The Lit Relayer is an [open-source service](https://github.com/LIT-Protocol/relay-server) currently hosted by Lit to facilitate onboarding into the Lit ecosystem. It helps reduce initial costs by covering or subsidizing certain interactions with the Lit network, such as minting PKPs and paying for network requests to perform actions like decryption, executing Lit Actions, and using PKPs.
+
+While the Relayer eases the onboarding process, it's important to note that its availability is not guaranteed. Users may experience rate limiting and/or congestion due to its shared nature.
+
+As you progress with your Lit integration, we recommend implementing Capacity Credits and/or the Payment Delegation Database in your application. These solutions offer more reliable and scalable options for managing your long-term usage of the Lit network.
```

Comment:
> As you progress with your Lit integration, we recommend implementing Capacity Credits and/or the Payment Delegation Database in your application. These solutions offer more reliable and scalable options for managing your long-term usage of the Lit network.

Consider: "As your application moves into production, we recommend implementing this functionality directly into your own application instead of using the Lit Relayer. This will ensure that you can mint / use Capacity Credits and Lit keys with minimal friction and disruptions in service, as your direct implementation will be much more reliable and scalable"

Reply:
Changed to:

As your application moves into production, we recommend implementing this functionality directly into your own application instead of using the Lit Relayer. This will ensure that you can use the Lit network with minimal friction and disruptions in service, as your direct implementation will be much more reliable and scalable.